java/ingest/ingest-runner/pom.xml

```diff
@@ -27,9 +27,16 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.amazonaws</groupId>
-            <artifactId>aws-java-sdk-cloudwatch</artifactId>
-            <version>${aws-java-sdk.version}</version>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>cloudwatch</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>sqs</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>dynamodb</artifactId>
```

Comment:
There's a header for "AWS version 2 dependencies" just below here. We also don't really need to declare the SQS and DynamoDB dependencies as they're brought in transitively.

We could move the CloudWatch dependency to the other AWS dependencies, and take the "version 2" out of the header since there are only version 2 dependencies now?